app/package.json

```diff
@@ -31,7 +31,6 @@
     "@veramo/message-handler": "^1.1.1-next.15",
     "@veramo/remote-client": "^1.1.1-next.15",
     "@veramo/selective-disclosure": "^1.1.1-next.15",
-    "@web3-react/authereum-connector": "^6.1.9",
```

Comment:
I removed this because we did not use it and it brings in an old "ethers.js" version